src/components/QuizView/QuizHeader.js

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react";
+import { Container, LinearProgress, makeStyles } from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+  header: {
+    display: "flex",
+    flexDirection: "column",
+    marginTop: "50px",
+    marginBottom: "30px",
+    "& h2, & p": {
+      textAlign: "center",
+      marginLeft: "auto",
+      marginRight: "auto",
+    },
+  },
+}));
+
+function QuizHeader({ categoryName, currentQuestionNumber, totalQuestions }) {
+  const classes = useStyles();
+  const [percentDone, setPercentDone] = useState(0);
```

Comment:
```suggestion
  const percentDone = (currentQuestionNumber / totalQuestions) * 100;
```

as mentioned [here](https://github.com/Soluto/taking-care-of-quizness-ui/pull/4/files#r581447654) I think we recompute `percentDone` on each re-render and avoid `useState` altogether here